ODIN_II/SRC/odin_ii.cpp

```diff
@@ -572,7 +572,7 @@ void get_options(int argc, char** argv) {
             global_args.verilog_files.value().size() > 0                             //have a verilog input list
         })) {
         parser.print_usage();
-        error_message(PARSE_ARGS, unknown_location, "%s", "Must include only one of either:\n\ta config file(-c)\n\ta blif file(-b)\n\ta verilog file(-V)\n");
+        warning_message(PARSE_ARGS, unknown_location, "%s", "Must include only one of either:\n\ta config file(-c)\n\ta blif file(-b)\n\ta verilog file(-V)\nUnless is used for infrastructure directly\n");
```

Comment:
this check should be moved later, to check we dont have both a blif and a verilog

so after the check on https://github.com/CAS-Atlantic/vtr-verilog-to-routing/pull/24/files#diff-e7605390c1f953098af6254c8fff55b01686e9db37e21d288b2b75d8e93644a6L269

do 
```
if (global_args.blif_file.provenance() == argparse::Provenance::SPECIFIED) {
    ...
```